calico-cloud_versioned_docs/version-18-2/threat/security-anomalies.mdx

```diff
@@ -0,0 +1,82 @@
+---
+description: Machine learning to automatically alert you when clusters have security issues.
+---
+
+# Detect and alert on anomalies
+
+:::note
+
+This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
+
+:::
+
+## Big picture
+
+Generate alerts for anomalous activity within a cluster.
+
+## Value
+
+The {{prodname}} anomaly detection engine analyzes patterns and alerts on potential
+threats and issues such as:
+
+- **Security issues** - threats like DGA, reconnaissance threats like IP sweep, but also issues with servers that can affect daily operations.
+
+- **Performance issues** - spikes in data transmission, and anomalous degradation in network communication that may impact application workloads.
+
+If there are any security or performance anomalies, you will automatically get alerts in Manager UI.
+
+## Features
+
+This how-to guide uses the following {{prodname}} features:
+
+- **Anomaly detection**
+
+## Concepts
+
+### About anomalies
+
+Anomaly detection uses {{prodname}} Elasticsearch logs ([flow](../visibility/elastic/flow/index.mdx) logs, [L7](../visibility/elastic/l7/index.mdx) logs, and [DNS](../visibility/elastic/dns/index.mdx) logs) to learn the behavior of cluster nodes, pods, services, and other entities that send log records (applications, load balancers, databases, etc.).
+
+The current Calico Cloud release includes the following anomaly detection algorithms.
+
+#### Domain Generation Algorithms (DGA)
+
+Looks for DNS requests for domain names that are likely to be created by [Domain Generation Algorithms (DGA)](https://en.wikipedia.org/wiki/Domain_generation_algorithm), frequently used by malware. Generated domain names (URLs) are used to communicate between the malware code and the malware's command and control servers. Presence of DGA may indicate presence of malware.
+
+## FAQ
```

Comment:
This is still good info, but consider moving it here so it fits more with the standard:

## Concepts

### About anomalies

Anomaly detections run unconditionally and are preconfigured with reasonable defaults that are optimized for frequency and severity. Anomaly detection uses {{prodname}} Elasticsearch logs ([flow](../visibility/elastic/flow/index.mdx) logs, [L7](../visibility/elastic/l7/index.mdx) logs, and [DNS](../visibility/elastic/dns/index.mdx) logs) to learn the behavior of cluster nodes, pods, services, and other entities that send log records (applications, load balancers, databases, etc.).